fenix/app/src/main/java/org/mozilla/fenix/HomeActivity.kt

```diff
@@ -662,7 +663,10 @@ open class HomeActivity : LocaleAwareAppCompatActivity(), NavHostActivity {
         components.core.pocketStoriesService.stopPeriodicSponsoredStoriesRefresh()
         privateNotificationObserver?.stop()
         components.notificationsDelegate.unBindActivity(this)
-        stopMediaSession()
+
+        if (this !is ExternalAppBrowserActivity) {
```

Comment:
this feels a little icky to have `HomeActivity` aware of its descendant types, but this is also a very silly QOL bug. Would this be cleaned up at all with some of your proposed RFC work?

Reply:
@MozillaNoah yep exactly this, noted in the PR desc 🙂 Essentially it will be the same as the [HomeActivity decoupling PR](https://github.com/mozilla-mobile/firefox-android/pull/4851) - now these checks are super explicit we can start refining the implementation

Reply:
My eyes completely missed the description, sorry. That all sounds good 👍

Reply:
No probs at all @MozillaNoah, appreciate the review 🙏